webapp/components/claim/components/email_to_ldap.jsx

```diff
@@ -104,13 +104,19 @@ export default class EmailToLDAP extends React.Component {
                             }}
                         />
                     </p>
+                    <input
+                        type='text'
+                        style={{display: 'none'}}
+                        name='fakeusernameremembered'
+                    />
```

Comment:
Chrome will try to autocomplete the first password and text field it finds based on some heuristics and it was causing the LDAP ID to auto-complete the email password field. Solution is based off this [stackoverflow thread](http://stackoverflow.com/a/15917221).